db/migrate/20170922144510_add_mail_to_addresses.rb

```diff
@@ -0,0 +1,17 @@
+class AddMailToAddresses < ActiveRecord::Migration[5.1]
+  def change
+    reversible do |direction|
+
+      direction.up do
+        # Only one address per company in DB
+        add_column :addresses, :mail, :boolean, default: true
+        change_column_default :addresses, :mail, false
+      end
+
+      direction.down do
+        remove_column :addresses, :mail
+      end
+
+    end
+  end
+end
```

Comment:
After reading up on this (I didn't know `reversible`), I wonder if the code below doesn't do the exactly same?

```
def change
  add_column :addresses, :mail, :boolean, default: true
end
```

Reply:
I don't think so.  The `up` migration does two things:

1) Creates the mail column and sets the value to `true` for all _existing_ addresses
2) Resets the default to `false` so any subsequent addresses are created that way.

Reply:
A, of course, I missed the setting it to "false". Sure sign I need coffee :)

My question was more about the `reversible`. I didn't know that Rails knows how to reverse most of the db commands, but I'm sure you do, so I wondered if there was a special reason to use `reversible`, and not just do:
```
def change
  add_column :addresses, :mail, :boolean, default: true
  change_column_default :addresses, :mail, from: true, to: false
end
```

Not asking you to change anything, just trying to learn.